templates/loginFailure.html

```diff
@@ -1,33 +1,34 @@
-      #message { background: white; max-width: 360px; margin: 100px auto 16px; padding: 32px 24px 8px; border-radius: 3px; }
-      #message h2 { color: #f44336; font-weight: bold; font-size: 16px; margin: 0 0 8px; }
-      #message h1 { font-size: 22px; font-weight: 300; color: rgba(0,0,0,0.6); margin: 0 0 16px;}
-      #message p { line-height: 140%; margin: 16px 0 24px; font-size: 14px; }
-      #message a { display: block; text-align: center; background: #039be5; text-transform: uppercase; text-decoration: none; color: white; padding: 16px; border-radius: 4px; }
-      #message, #message a { box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); }
-      #load { color: rgba(0,0,0,0.4); text-align: center; font-size: 13px; }
-      @media (max-width: 600px) {
-        body, #message { margin-top: 0; background: white; box-shadow: none; }
-        body { border-top: 16px solid #f44336; }
-      }
-      code { background: #37474f; color: white; border-radius: 2px; padding: 2px 4px; font-size: 12px; margin: 0 4px; }
-    </style>
-  </head>
-  <body>
-    <div id="message">
-      <h2>Oops!</h2>
-      <h1>Firebase CLI Login Failed</h1>
-      <p>The Firebase CLI login request was rejected or an error occurred. Please
-      run <code>firebase login</code> again or contact support if you continue
-      to have difficulty logging in.</p>
-    </div>
-  </body>
+  <style media="screen">
+    body { background: #ECEFF1; color: rgba(0,0,0,0.87); font-family: Roboto, Helvetica, Arial, sans-serif; margin: 0; padding: 0; }
+    #message { background: white; max-width: 360px; margin: 100px auto 16px; padding: 32px 24px 8px; border-radius: 3px; }
+    #message h2 { color: #f44336; font-weight: bold; font-size: 16px; margin: 0 0 8px; }
+    #message h1 { font-size: 22px; font-weight: 300; color: rgba(0,0,0,0.6); margin: 0 0 16px;}
+    #message p { line-height: 140%; margin: 16px 0 24px; font-size: 14px; }
+    #message p a { background: none; display: inline; box-shadow: none;  color:#039BE5; text-transform: none; padding: 0}
+    #message a{ display: block; text-align: center; background: #039be5; text-transform: uppercase; text-decoration: none; color: white; padding: 16px; border-radius: 4px; }
+    #message, #message a { box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); }
+    #load { color: rgba(0,0,0,0.4); text-align: center; font-size: 13px; }
+    @media (max-width: 600px) {
+      body, #message { margin-top: 0; background: white; box-shadow: none; }
+      body { border-top: 16px solid #f44336; }
+    }
+    code { background: #37474f; color: white; border-radius: 2px; padding: 2px 4px; font-size: 12px; margin: 0 4px; }
+  </style>
+</head>
+<body>
+<div id="message">
+  <h2>Oops!</h2>
+  <h1>Firebase CLI Login Failed</h1>
+  <p>The Firebase CLI login request was rejected or an error occurred. Please
```

Comment:
It's not a big deal but this PR makes it look like you changed every line in this file despite only changing one line (I think).  Did you autoformat it through your editor or something like that?

Reply:
@samtstern 
You're right for some reason my IDE autoformat the entire file.

Reply:
IMO, if you undo the formatting of the entire file and scope the change down to the link, I think this would be an acceptable PR. (also, update it with the `master` branch - sorry we're so delayed on this response)